graph-subscriptions-api/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM rust:1.68-bullseye AS build
+
+# used to pull private github packages needed in the build
+ARG GH_USER
+ARG GH_TOKEN
```

Comment:
Are there private dependencies? I didn't notice any. If not, we can remove this and the GH credentials setup stuff below.

Reply:
It has a dep on the `graph-subscriptions` lib that it requests through github. Since this is a private repo, I assume so

Reply:
is `toolshed` private?

Reply:
Nope, all public

Reply:
okay perfect. thanks!